<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>750</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BF - Bambook Factory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bf.qrc">
    <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionCopyToBambook"/>
   <addaction name="actionCopyFromBambook"/>
   <addaction name="actionReadTheBook"/>
   <addaction name="actionDeleteTheBook"/>
   <addaction name="actionOptions"/>
   <addaction name="actionOpenTheFolder"/>
   <addaction name="actionImportPDF"/>
   <addaction name="actionTxtImport"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/Disconnect64.png</normaloff>
     <normalon>:/images/Connect64.png</normalon>
     <disabledoff>:/images/Disconnect64.png</disabledoff>
     <disabledon>:/images/Connect64.png</disabledon>:/images/Disconnect64.png</iconset>
   </property>
   <property name="text">
    <string>Connect to Bambook</string>
   </property>
   <property name="toolTip">
    <string>Connect to Bambook</string>
   </property>
  </action>
  <action name="actionCopyToBambook">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/CopyToBambook64.png</normaloff>:/images/CopyToBambook64.png</iconset>
   </property>
   <property name="text">
    <string>Upload to Bambook</string>
   </property>
   <property name="toolTip">
    <string>Upload the book to Bambook</string>
   </property>
  </action>
  <action name="actionCopyFromBambook">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/CopyFromBambook.png</normaloff>:/images/CopyFromBambook.png</iconset>
   </property>
   <property name="text">
    <string>Download from Bambook</string>
   </property>
   <property name="toolTip">
    <string>Download the book from Bambook</string>
   </property>
  </action>
  <action name="actionReadTheBook">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/ReadBook64.png</normaloff>:/images/ReadBook64.png</iconset>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
   <property name="toolTip">
    <string>Read the book</string>
   </property>
  </action>
  <action name="actionDeleteTheBook">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/DeleteBook64.png</normaloff>:/images/DeleteBook64.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>delete the book</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/options64.png</normaloff>:/images/options64.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options and tools</string>
   </property>
  </action>
  <action name="actionOpenTheFolder">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/folder64.png</normaloff>:/images/folder64.png</iconset>
   </property>
   <property name="text">
    <string>Open the folder</string>
   </property>
   <property name="toolTip">
    <string>Open the folder</string>
   </property>
  </action>
  <action name="actionImportPDF">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/pdf.png</normaloff>
     <disabledoff>:/images/pdf.png</disabledoff>:/images/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Import PDF</string>
   </property>
   <property name="toolTip">
    <string>Import PDF files</string>
   </property>
  </action>
  <action name="actionTxtImport">
   <property name="icon">
    <iconset resource="bf.qrc">
     <normaloff>:/images/txt.png</normaloff>
     <disabledoff>:/images/txt.png</disabledoff>:/images/txt.png</iconset>
   </property>
   <property name="text">
    <string>Split TXT and import</string>
   </property>
   <property name="toolTip">
    <string>Split TXT and import to Bambook</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="bf.qrc"/>
 </resources>
 <connections/>
</ui>
